Grammar fix (#5326)
flutter/website: src/docs/resources/faq.md

### How does Flutter run my code on iOS? {#run-ios}

> The engine's C and C++ codes are compiled with LLVM. The Dart code (both the

Reviewer: ```suggestion
The engine's C and C++ code is compiled with LLVM. The Dart code (both the
```
